apigee-cf-service-broker/api/service_instances.js

```diff
@@ -120,6 +120,32 @@ function actionValidate(req, res, next) {
   }
 }
 
+function deriveProtocol(params){
+  //default to https if no protocol is provided
+  const proto = params.protocol ? params.protocol.toString().trim().toLowerCase() : 'https'
+  const ret = {
+    errors: []
+  }
+  if (proto === "http" || proto === "https"){
+    ret.protocol = proto
+  }
+  else if (params.protocol){
+    ret.errors.push(params.protocol.toString().split(/[\s,]+/))
```

Comment:
`split` returns an array, so `push` will create a nested array. Various ways of combining arrays in JavaScript, depending on the version, but since `errors` is always empty at this point, simpler to just reassign it.